ops/services/web_application_firewall/main.tf

```diff
@@ -78,10 +86,33 @@ resource "azurerm_web_application_firewall_policy" "sr_waf_policy" {
       selector_match_operator = "Equals"
     }
 
+    exclusion {
+      match_variable          = "RequestArgNames"
+      selector                = "iss"
```

Comment:
I saw in your PR request that `street` and `namePrefixMatch` are an excepted argument value for analysis. What is `iss` for? 🤔

Reply:
`iss` is a query string argument that comes from Okta's authentication process. WAF is erroneously interpreting this as a cross-site URL injection, when it's actually a required parameter that is passed in whenever someone logs into SimpleReport via Okta. If we don't exclude the parameter, nobody will be able to log in once we turn this on. 😓 

I hope that helps! I'm specifying a selector-based exclusion for just this query argument, so we don't have to turn the rule off altogether. This approach is a bit more surgical.

Reply:
Great question!

In Azure, you can see what the WAF is up to by navigating to the Application Gateway logs, and running the following query:

```
AzureDiagnostics 
| where ResourceProvider == "MICROSOFT.NETWORK" and Category == "ApplicationGatewayFirewallLog"
```

When this gets run, it will pull up everything that the WAF has alerted on over the past 24 hours, including what rule was tripped, and what specifically tripped it. This query string argument noted the Okta endpoint that was being hit, which is a URL. WAF _freaks_ when it sees a URL embedded as a query string argument, so it flagged that item specifically.